src/sentry/static/sentry/app/components/events/eventCauseEmpty.jsx

```diff
@@ -242,6 +243,33 @@ const DismissButton = styled(Button)`
   border-bottom-left-radius: 0;
 `;
 
+const ExampleCommitPanel = styled(Panel)`
+  overflow: hidden;
+  pointer-events: none;
+  position: relative;
+  padding-right: ${space(3)};
+
+  &:after {
+    display: block;
+    content: 'Example';
+    position: absolute;
+    top: ${space(2)};
```

Comment:
I wouldn't use `space` here, since it just happens to be the right pixel value, if we were to adjust spacing in `space` this would probably end up causing this to misalign right?